### PR TITLE
testing/amule: upgrade to 11015

### PR DESCRIPTION
**Pull request by maxice8**
- Switch to wxgtk

### CODE REVIEW

**Comment by maxice8** [comment]
@AmatCoder

File: testing/amule/APKBUILD [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
 subpackages="$pkgname-lang $pkgname-doc"
-source="http://amule.sourceforge.net/tarballs/${_pkgname}${pkgver}.tar.bz2"
-builddir=$srcdir/${_pkgname}${pkgver}
+source="$pkgname-$pkgver.tar.gz::https://repo.or.cz/amule.git/snapshot/$_commit.tar.gz"
 
-prepare() {
-	cd "$builddir"
-	update_config_sub
-}
+builddir="$srcdir/$pkgname-${_commit:0:7}"
```
Comment (tcely) [inline testing/amule/APKBUILD]:
```suggestion
builddir="$srcdir/$pkgname-${_commit%${_commit#???????}}"
```
Remove bashism

File: testing/crypto++/APKBUILD [hunk 2]
```diff
@@ -4,33 +4,36 @@ pkgname=crypto++
...
+source="https://www.cryptopp.com/${_pkgname}${_pkgver}.zip
+	crypto++.pc
+	"
 
 builddir="$srcdir"
 
 build() {
-	cd "$builddir"
-	sed -i -e 's/^CXXFLAGS/#CXXFLAGS/' GNUmakefile || return 1
+	sed -i -e 's/^CXXFLAGS/#CXXFLAGS/' GNUmakefile
```
Comment (tcely) [inline testing/crypto++/APKBUILD]:
```suggestion
	sed -i -e '/^CXXFLAGS/s/^/#/' GNUmakefile
```

File: testing/crypto++/APKBUILD [hunk 3]
```diff
@@ -4,33 +4,36 @@ pkgname=crypto++
...
-	cd "$builddir"
-	install -Dm644 libcryptopp.so "$pkgdir"/usr/lib/libcryptopp.so || return 1
-	install -Dm644 License.txt "$pkgdir"/usr/share/licenses/$pkgname/LICENSE || return 1
-	mkdir -p "$pkgdir"/usr/include/cryptopp/ || return 1
-	install -m644 *.h "$pkgdir"/usr/include/cryptopp/ || return 1
+	make install DESTDIR="$pkgdir" PREFIX=/usr
+	rm -rf "$pkgdir"/usr/bin "$pkgdir"/usr/share/cryptopp
+
+	install -Dm644 "$srcdir"/crypto++.pc "$pkgdir"/usr/lib/pkgconfig/crypto++.pc
+	sed -e "s,@@VERSION@@,${version},g" -i "$pkgdir"/usr/lib/pkgconfig/libcrypto++.pc
```
Comment (tcely) [inline testing/crypto++/APKBUILD]:
```suggestion
	sed -e 's,@@VERSION@@,${version},g' -i "$pkgdir"/usr/lib/pkgconfig/libcrypto++.pc
```

version is not set earlier, so I guess you don't want to expand to empty here.

File: testing/crypto++/APKBUILD [hunk 4]
```diff
@@ -4,33 +4,36 @@ pkgname=crypto++
 _pkgname=cryptopp
 pkgver=5.6.5
```
Comment (tcely) [inline testing/crypto++/APKBUILD]: This version is fairly old now.

https://www.cryptopp.com/downloads.html#CRYPTOPP_820

**Comment by maxice8** [comment]
this now depends the crypto++ PR #8441

**Comment by AmatCoder** [comment]
@maxice8 

Thanks for upgrading those packages.
I have no time to play with Alpine Linux lately so you can take maintenership if you wants.

Regards.

**Pull request closed without merging**